src/categories/categories.service.ts

```diff
@@ -57,10 +57,12 @@ export class CategoriesService {
   async getAllApis(categoryId: string){
     try {
       //check if category exists
-      const categoryExists = await this.categoryRepo.findOne({where:{id:categoryId}})
+      const category = await this.categoryRepo.findOne({
+        where:{id:categoryId},
+        relations:{api: true}})
```

Comment:
please jefferey why did u add the relations property?

Reply:
@alahirajeffrey

Reply:
@PhilemonBrain I made the changes we spoke about on the call and pushed it. I then decided to test it out again and its not working again

Reply:
@Brain I think the problem was that I switched branches and tried testing again.